sdk/core/core-auth/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.package",
```

Comment:
or should I update `../../../tsconfig.package` so all have the default target of `es5`?

Reply:
I just did that.

Reply:
Since only some packages have a need for IE11 support, only core packages need to target ES5. Non-core should remain ES6 at a minimum. ES5 emit is massive relative to ES6 emit for things like async functions, and so should be avoided unless absolutely necessary.

Reply:
Got it. Will change back to fix core-auth only

Reply:
Done